Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -35,6 +35,8 @@
       - id: LPPBoxFolderSecDoc
         prob: 0.5
       - id: LPPHOSUplinkShop #На будущее, будет хайриском
+      - id: LPPRubberStampConfidentional #LPP
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)